src/auto/constant_expression.rs

```diff
@@ -0,0 +1,48 @@
+// This file was generated by gir (https://github.com/gtk-rs/gir)
+// from gir-files (https://github.com/gtk-rs/gir-files)
+// DO NOT EDIT
+
+use glib;
+use glib::object::Cast;
+use glib::translate::*;
+use gtk_sys;
+use std::fmt;
+use Expression;
+
+glib_wrapper! {
+    pub struct ConstantExpression(Object<gtk_sys::GtkConstantExpression, ConstantExpressionClass>) @extends Expression;
+
+    match fn {
+        get_type => || gtk_sys::gtk_constant_expression_get_type(),
+    }
+}
+
+impl ConstantExpression {
+    //pub fn new(value_type: glib::types::Type, : /*Unknown conversion*//*Unimplemented*/Fundamental: VarArgs) -> ConstantExpression {
+    //    unsafe { TODO: call gtk_sys:gtk_constant_expression_new() }
+    //}
+
+    pub fn new_for_value(value: &glib::Value) -> ConstantExpression {
```

Comment:
This should probably get some addition in gir (just create an issue for now). Similar to `new_with_XXX()`, this should here become simply `for_value()` without the `new_` prefix.

Reply:
Done https://github.com/gtk-rs/gir/issues/974